Library/ViewModels/ManageViewPledgeViewModel.swift

```diff
@@ -77,7 +79,7 @@ public final class ManageViewPledgeViewModel:
     self.viewDidLoadObserver.send(value: ())
   }
 
-  public let configurePaymentMethodView: Signal<Project, Never>
+  public let configurePaymentMethodView: Signal<GraphUserCreditCard.CreditCard, Never>
```

Comment:
Similar to my comment here https://github.com/kickstarter/ios-oss/pull/835#discussion_r328359310

Would communicate the change with Christella to prevent from merge conflicts

Reply:
So was the typealias created yet? I think that creating it in this PR will add a ton of new diffs, though.